arches/app/src/components/ControlledListManager/SidepanelDataView.vue

```diff
@@ -0,0 +1,254 @@
+<script setup lang="ts">
```

Comment:
is this level of abstraction necessary? I'm not sold either way but we should chat about it